certified-connectors/Knowledgeone RecFind 6/apiProperties.json

```diff
@@ -0,0 +1,31 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Security Key",
+          "description": "The Security Key for this api",
+          "tooltip": "Provide your Security Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FB8003",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "HostUrl",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@queryParameters('HostUrl')"
+        }
+      }
+    ],
+    "publisher": "Knowledgeone Corporation"
```

Comment:
@ecardakaris-knowledgeonecorp Please go thru the instructions mentioned in the documentation to add metadata for your connector.

Please add stackOwner property.